app/helpers/xls/tab_details.py

```diff
@@ -0,0 +1,217 @@
+from app.helpers.time import to_fr_tz
+from app.helpers.xls.common import (
+    write_sheet_header,
+    light_grey_hex,
+    light_yellow_hex,
+    light_blue_hex,
+    light_green_hex,
+    light_red_hex,
+    formats,
+    write_tab_headers,
+    write_cells,
+    merge_cells_if_needed,
+)
+from app.models.activity import Activity
+from app.templates.filters import format_activity_type
+
+
+def write_day_details_sheet(
+    wb, wdays_by_user, require_mission_name, companies, min_date, max_date
+):
+    sheet = wb.add_worksheet("Détails")
```

Comment:
I thought it would be more convenient to remove it but after a discussion with the team, we'll leave it to prevent file modification that would alter its integrity in case of control.